auto_nag/scripts/vacant_team_manager.py

```diff
@@ -0,0 +1,84 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from typing import List
+
+from libmozdata.bugzilla import BugzillaProduct
+
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag.team_managers import TeamManagers
+
+
+class TeamManagerVacant(BzCleaner):
+    def __init__(self) -> None:
+        super(TeamManagerVacant, self).__init__()
+        self.query_url = None
+        self.inactive_weeks_number = self.get_config("inactive_weeks_number", 26)
```

Comment:
```suggestion
```
This is not used. We do not check activity.

Reply:
Removed in d838868542720999e659d2e9cb1fb2ee6c399e27.